features/foutafhandeling.feature

```diff
@@ -152,7 +21,6 @@ Rule: type van parameter waarde is correct
 
         Voorbeelden:
         | path      | query string                                          | detail                                                                                           | parameter | code             | reason omschrijving                   |
-        | /adressen | ?pandIdentificatie=0345100002016017&page=a&pageSize=3 | Ongeldige waarde [a] opgegeven voor parameter [page]                                             | page      | integer          | Waarde is geen geldige integer.       |
         | /panden   | ?locatie=abc,def                                      | Ongeldige waarde [abc,def] opgegeven voor parameter [locatie]                                    | locatie   | number           | Waarde is geen geldig decimaal getal. |
```

Comment:
Deze rule kan volgens mij vervallen en dit scenario kan onder 'Rule: waarde is valide' worden geplaatst.
In alle gevallen is de waarde onjuist t.o.v. wat er is gespecificeerd. Dit kan zijn omdat: 
1. de waarde van een ander type is dan in de specificatie staat
2. de waarde niet overeenkomt met het opgegeven CRS
3. de waarde kleiner is dan een gespecificeerde minimum waarde
4. de waarde groter is dan een gespecificeerde maximum waarde
M.i. hoeft er dan geen onderscheid te zijn tussen 1 en de rest. Valide betekent m.i. dat de waarde voldoet aan de specificaties en niet valide dus niet. Waarom moet dit onderscheid er zijn?

Reply:
Je kan de voorgestelde wijzigingen het beste zelf doorvoeren. Dit is niet meer mijn P.R. aangezien er andere scenario's door jou zijn toegevoegd. Verder weet ik ook niet welke code en reason tekst je hiervoor wilt gebruiken.